oakpal-checks/pom.xml

```diff
@@ -266,27 +266,38 @@
     </profiles>
 
     <dependencies>
+        <dependency>
+            <groupId>org.jetbrains</groupId>
```

Comment:
@adamcin what's this dep for? Is this IDEA specific?

Reply:
It's just for `@NotNull` and `@Nullable`. Not IDEA specific, necessarily, but I haven't tested it with Eclipse, for example. Anyways, I chose this for nullability annotations purely because it's what is used in the jackrabbit-oak and filevault projects and it works for me (/shrug). I think jackrabbit switched away from the javax annotations to avoid the runtime package dependency and the whole JEE trademark mess.